pants.ini

```diff
@@ -388,11 +399,28 @@ verify_commit: False
 
 
 [python-setup]
+interpreter_search_paths: [
+     '<PEXRC>',
+     '<PATH>'
+     # TODO(#7735): We need to add this entry for remoting to be able to discover a valid
+     # interpreter, because <PATH> will refer to the host PATH and not the remote value. This value
+     # was found by inspecting the docker image for remoting.
+     "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/python3.6/bin:/usr/local/go/bin",
+  ]
 interpreter_constraints: ["CPython>=3.6,<4"]
 interpreter_cache_dir: %(pants_bootstrapdir)s/python_cache/interpreters
 resolver_cache_dir: %(pants_bootstrapdir)s/python_cache/requirements
 
 
+[python-native-code]
```

Comment:
I would feel better about all of this if we put all of these options in a separate file which can be loaded at use-time. Maybe `pants.remote.ini`.

Reply:
Great idea!